install/resources/mybb_theme.xml

```diff
@@ -3966,14 +3966,14 @@ var announcement_quickdelete_confirm = "{$lang->announcement_quickdelete_confirm
 {$forums}
 </table>
 <br />]]></template>
-		<template name="forumdisplay_thread" version="1813"><![CDATA[<tr class="inline_row">
+		<template name="forumdisplay_thread" version="1823"><![CDATA[<tr class="inline_row">
 	<td align="center" class="{$bgcolor}{$thread_type_class}" width="2%"><span class="thread_status {$folder}" title="{$folder_label}">&nbsp;</span></td>
 	<td align="center" class="{$bgcolor}{$thread_type_class}" width="2%">{$icon}</td>
 	<td class="{$bgcolor}{$thread_type_class}">
 		{$attachment_count}
 		<div>
-          <span>{$prefix} {$gotounread}{$thread['threadprefix']}<span class="{$inline_edit_class} {$new_class}" id="tid_{$inline_edit_tid}"><a href="{$thread['threadlink']}">{$thread['subject']}</a></span>{$thread['multipage']}</span>
-			<div class="author smalltext">{$thread['profilelink']}</div>
+          <span>{$prefix} {$gotounread}<a href="{$mybb->settings['bburl']}/forumdisplay.php?fid={$mybb->input['fid']}&prefix={$thread['prefix']}">{$thread['threadprefix']}</a><span class="{$inline_edit_class} {$new_class}" id="tid_{$inline_edit_tid}"><a href="{$thread['threadlink']}">{$thread['subject']}</a></span>{$thread['multipage']}</span>
+			<div class="author smalltext">{$thread['profilelink']} <span class="smalltext" title="{$lang->thread_started_time_date} {$threaddate}, {$threadtime}">({$threaddate}, {$threadtime})</span></div>
```

Comment:
This link should be built using core functions, i.e. Google SEO won't be able to replace this hard coded link.

Reply:
Actually, I think this should probably be a new template, becase the link code will be present even if the thread has no prefix.

Reply:
Actually, I think this should probably be a new template, becase the link code will be present even if the thread has no prefix.

Reply:
Ok, but that would be quite tricky... we do not even have a separate template for a thread prefix in forumdisplay, right?

Reply:
> This link should be built using core functions, i.e. Google SEO won't be able to replace this hard coded link.

But do core functions even currently support getting a forum link with a query parameter like "prefix"? It doesn't look like it to me. It looks as though that support would have to be added, which probably is beyond the scope of this PR?

Reply:
> But do core functions even currently support getting a forum link with a query parameter like "prefix"? It doesn't look like it to me. It looks as though that support would have to be added, which probably is beyond the scope of this PR?

It has a parameter for filtering prefixes in this way `/forumdisplay.php?fid={fid}&prefix={prefix_id}`.

Reply:
> It has a parameter for filtering prefixes in this way `/forumdisplay.php?fid={fid}&prefix={prefix_id}`.

Perhaps we're miscommunicating. The core function to which I'm referring is `get_forum_link()` in inc/functions.php. The two parameters to that function are `$fid` and `$page`. I see no scope for a prefix in those parameters.

Reply:
> > It has a parameter for filtering prefixes in this way `/forumdisplay.php?fid={fid}&prefix={prefix_id}`.
> 
> Perhaps we're miscommunicating. The core function to which I'm referring is `get_forum_link()` in inc/functions.php. The two parameters to that function are `$fid` and `$page`. I see no scope for a prefix in those parameters.

Oh I see. But `&prefix=` can be always appended to a URL formed by `get_forum_link()` or what ever if it's a link of current viewing forum.

Reply:
> > Perhaps we're miscommunicating. The core function to which I'm referring is `get_forum_link()` in inc/functions.php. The two parameters to that function are `$fid` and `$page`. I see no scope for a prefix in those parameters.
> 
> Oh I see. But `&prefix=` can be always appended to a URL formed by `get_forum_link()` or what ever if it's a link of current viewing forum.

Sorry, inaccurate comment was made.

The forum URL has to be formed as a variable maybe `$forum_url` just like how `$page_url` was formed, except that there's no other parameter, so that in the template we could use `{$forum_url}?prefix={$prefixid}` as the link href. 

Note `{$prefixid}` should be changed to the var accordingly. Maybe `bburl` should be added as well?

Reply:
> in the template we could use `{$forum_url}?prefix={$prefixid}`

Yes. But only sometimes. Other times we would need that `?` to be an `&amp;`, especially when basic URLs rather than core/SEO-plugin SEO-friendly URLs were operative.

Reply:
So, I guess that's really a "No, it's more complicated than that".

Reply:
I think that the best solution would be introduce a new function to build links with a set of parameters. Any other manual way of trying to do it is sure to cause problems.